lib/externalGraderResults.js

```diff
@@ -112,18 +117,25 @@ function processMessage(data, callback) {
         return callback(error.makeWithData('Message does not contain a valid grading job id.', data));
     }
 
+    var logBundle = [];
+    var loggingTimeout = setTimeout(logAtTimeout, 10000, jobId, logBundle);
+
+    logBundle.push({ timestamp: Date.now(), msg: `externalGraderResults: ${jobId} processMessage ${data}`});
```

Comment:
I suggest just logging data as actual objects, rather than as strings. Also no need to re-log `jobId`.
```suggestion
    logBundle.push({ timestamp: Date.now(), msg: 'processMessage', data});
```